### PR TITLE
Add Telus

### PR DESCRIPTION
**Pull request by jpayoung**
Add entry Bell in _data/services.yml
- Icon
- No 2FA

### CODE REVIEW

**Comment by jpayoung** [comment]
Looks like I don't know my alphabet! Fixing now.

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
The use a rather bad logo but I think this one is the best one:
telus
It doesn't contain any text.

File: _data/utilities.yml [hunk 1]
```diff
@@ -328,6 +328,12 @@ websites:
     facebook: Telstra24x7
     img: telstra.png
 
+  - name: Telus
+    url: https://www.telus.com
+    twitter: Telus
```
Comment (Carlgo11) [inline _data/utilities.yml]:
```suggestion
    twitter: TELUSSupport
```

File: _data/utilities.yml [hunk 2]
```diff
@@ -328,6 +328,12 @@ websites:
...
+    facebook: TELUSSupport
```
Comment (Carlgo11) [inline _data/utilities.yml]:
```suggestion
    facebook: telus
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: _data/utilities.yml [hunk 3]
```diff
@@ -328,6 +328,12 @@ websites:
     facebook: Telstra24x7
     img: telstra.png
 
+  - name: Telus
+    url: https://www.telus.com
+    twitter: TELUSSupport
```
Comment (Carlgo11) [inline _data/utilities.yml]: 🤔 
```suggestion
    twitter: TELUSsupport
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — APPROVED** [review]
Thanks! 😄

**Review by kmpoppe — APPROVED** [review]

**Pull request merged**